app/util/math.spec.ts

```diff
@@ -59,6 +59,20 @@ describe('with default locale', () => {
   ])('splitDecimal %d -> %s', (input, output) => {
     expect(splitDecimal(input)).toEqual(output)
   })
+
+  it.each([
+    [0n, '0'],
+    [1n, '1'],
+    [155n, '155'],
+    [999999n, '999,999'],
+    [9999999n, '10M'],
+    [492038458320n, '492B'],
+    [894283412938921, '894.3T'],
+    [1293859032098219, '1.3E15'],
+    [23094304823948203952304920342n, '23.1E27'],
+  ])('displayBigNum %d -> %s', (input, output) => {
+    expect(displayBigNum(input)).toEqual(output)
+  })
```

Comment:
these bigint literals are the reason for the `target: es2020` bump in the tsconfig